lgc/CMakeLists.txt

```diff
@@ -111,10 +127,12 @@ endif()
 target_include_directories(LLVMlgc
 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/interface>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/interface>
```

Comment:
Presumably this is so that auto-generated dialect file(s) become part of the LGC interface. Why? (And is this separate from the idea that the LLPC front-end might also use a dialect in the future?)

Reply:
The idea is that LLPC code can generate ops using `builder.create<FooOp>`.